packages/dispatcher-converter/src/util/FolderOperations.js

```diff
@@ -34,11 +35,7 @@ class FolderOperationsUtility {
         // if is directory
         if (fs.existsSync(dirPath) && fs.lstatSync(dirPath).isDirectory()) {
             try {
-                fs.rmdirSync(dirPath, { recursive: true }, (err) => {
-                    if (err) {
-                        logger.error("Error while deleting " + dirPath);
-                    }
-                });
+                rimraf.sync(dirPath);
```

Comment:
fs.rmdirSync had issues deleting non-empty directories and symlinks. rimraf is a better option to guarantee the deletion.

Reply:
@dichaudhary we already have a _util_ method for recursive deletion - [deleteFolderRecursive](https://github.com/adobe/aem-cloud-service-source-migration/blob/master/packages/commons/src/util.js#L38). Can you please use that?

Reply:
correct ! my bad ! Thanks, @ManasMaji, just did the changes.